src/sentry/static/sentry/app/components/charts/components/xAxis.jsx

```diff
@@ -1,17 +1,27 @@
 import {getFormattedDate} from 'app/utils/dates';
 import theme from 'app/utils/theme';
 
-import {truncationFormatter} from '../utils';
+import {truncationFormatter, useShortInterval} from '../utils';
 
 export default function XAxis({
   isGroupedByDate,
-  shouldRenderTimeOnly,
+  useShortDate,
+  axisLabel,
+  axisTick,
+  axisLine,
+
+  start,
+  end,
+  period,
   utc,
   ...props
 } = {}) {
-  const axisLabelFormatter = value => {
+  const axisLabelFormatter = (value, index) => {
     if (isGroupedByDate) {
-      const format = shouldRenderTimeOnly === 'hour' ? 'LT' : 'MMM Do';
+      const dateFormat = useShortDate ? 'MMM Do' : 'MMM D LT';
```

Comment:
Having this here will is great. Will help make all our charts more consistent. Should we also take the user's clock preferences into account when formatting axis labels?

Reply:
Yeah, I'll do followups for clock and maybe tz preferences as well.